cypress/integration/map.controls.js

```diff
@@ -18,7 +18,7 @@ describe('Map controls', () => {
       expect(
         mapElt
           .getStyle()
-          .layers.find(({ id }) => id === '467f57e3ed81809ad58f090426a3893a'),
+          .layers.find(({ id }) => id === '5b0371ebc5d25ba6f7ce7068a8a959a7'),
```

Comment:
Shouldn't this string be in a variable (with a proper comment) so we know where it comes from? (and how to update it)

Reply:
cc @jrmi

Reply:
Done.

Reply:
Perfect! 👍